rt/rs/security/sso/oidc/src/main/java/org/apache/cxf/rs/security/oidc/idp/OidcUserSubject.java

```diff
@@ -54,6 +55,7 @@ public OidcUserSubject(UserSubject sub) {
 
     @Lob
     @Basic(fetch = FetchType.EAGER)
+    @Column(length = 1024)
```

Comment:
Where does 1024 come from?

Reply:
Looks like recent Hibernate cause [1]: 

> creates column as CLOB(255) 

So I set some higher value.

1. https://stackoverflow.com/a/26805926/3476615

Reply:
I'm wondering if a JWT token can be longer than 1024 characters....say if it has a lot of claims?

Reply:
updated @Column annotation: explicit 'blob' column type without length